docs/userguide/containers/dockerrepos.md

```diff
@@ -42,6 +42,7 @@ You can now commit and push your own images up to your repos on Docker Hub.
 > **Note:**
 > Your authentication credentials will be stored in the `~/.docker/config.json`
 > authentication file in your home directory.
+> For Windows users, this Docker file is stored in `$HOME/.docker/config.json`
```

Comment:
@lirantal  Thank you for the contribution. If the path for all OSs is `~/.docker/config.json` then the existing note is true for all users. I guess I'm missing why we need to mention Windows in particular.

BTW, it isn't a "Docker file" it is a configuration file. `Dockerfile` is a keyword. It is the Engine configuration file.